src/app/countup/countup.directive.ts

```diff
@@ -98,7 +98,7 @@ export class CountUpDirective implements AfterViewInit {
     const up = (end > sta) ? 1 : -1;
```

Comment:
Hi @Hrafven, I think the issue is with my logic here. The expression is reversed. It should be

` const up = (end > sta) ? -1 : 1;`

instead of

` const up = (end > sta) ? 1 : -1;`

Because this needs to work for counting in both directions.